src/Microsoft.VisualStudio.ProjectSystem.Managed.VS.UnitTests/GlobalSuppressions.cs

```diff
@@ -10,3 +10,5 @@
 [assembly: System.Diagnostics.CodeAnalysis.SuppressMessage("Style", "IDE0018:Inline variable declaration", Justification = "<Pending>", Scope = "member", Target = "~M:Microsoft.VisualStudio.Shell.Interop.IVsHierarchyFactory.ImplementGetProperty(Microsoft.VisualStudio.Shell.Interop.IVsHierarchy,Microsoft.VisualStudio.Shell.VsHierarchyPropID,System.Int32)")]
 [assembly: System.Diagnostics.CodeAnalysis.SuppressMessage("Style", "IDE0018:Inline variable declaration", Justification = "<Pending>", Scope = "member", Target = "~M:Microsoft.VisualStudio.Shell.Interop.IVsHierarchyFactory.ImplementGetProperty(System.Int32)~Microsoft.VisualStudio.Shell.Interop.IVsHierarchy")]
 [assembly: System.Diagnostics.CodeAnalysis.SuppressMessage("Style", "IDE0018:Inline variable declaration", Justification = "<Pending>", Scope = "member", Target = "~M:Microsoft.VisualStudio.Shell.Interop.IVsProjectFactory.ImplementOpenItemWithSpecific(Microsoft.VisualStudio.Shell.Interop.IVsProject4,System.Guid,System.Guid,System.Int32)")]
+
```

Comment:
I think the fixer tries to scan existing file and adds an extra new line only for first attribute - so seems like a minor bug to me. If you have a self contained repro, can you file a bug?